configFileProductions/spring2024ConfigFiles/michele/wzp6_ee_nunuH_Hbb_ecm240.sin

```diff
@@ -79,13 +79,13 @@ $shower_method          = "PYTHIA6"
 
 
 
-$ps_PYTHIA_PYGIVE = "MDME(210,1)=0; MDME(211,1)=0; MDME(212,1)=0; MDME(213,1)=0; MDME(214,1)=1; MDME(215,1)=0; MDME(218,1)=0; MDME(219,1)=0; MDME(220,1)=0; MDME(222,1)=0; MDME(223,1)=0; MDME(224,1)=0; MDME(225,1)=0; MDME(226,1)=0; MSTJ(28)=0; PMAS(25,1)=125.; PMAS(25,2)=0.4143E-02; MSTJ(41)=2; MSTU(22)=2000; PARJ(21)=0.40000; PARJ(41)=0.11000; PARJ(42)=0.52000; PARJ(81)=0.25000; PARJ(82)=1.90000; MSTJ(11)=3; PARJ(54)=-0.03100; PARJ(55)=-0.00200; PARJ(1)=0.08500; PARJ(3)=0.45000; PARJ(4)=0.02500; PARJ(2)=0.31000; PARJ(11)=0.60000; PARJ(12)=0.40000; PARJ(13)=0.72000; PARJ(14)=0.43000; PARJ(15)=0.08000; PARJ(16)=0.08000; PARJ(17)=0.17000; MSTP(3)=1;MSTP(71)=1;  MSTP(151)=1; PARP(151)=0.0098; PARP(152)=2.54e-5; PARP(153)=0.646; PARP(154)=1.937; MSTJ(22)=4; PARJ(73)=2250; PARJ(74)=2500"
+$ps_PYTHIA_PYGIVE = "MDME(210,1)=0; MDME(211,1)=0; MDME(212,1)=0; MDME(213,1)=0; MDME(214,1)=1; MDME(215,1)=0; MDME(218,1)=0; MDME(219,1)=0; MDME(220,1)=0; MDME(222,1)=0; MDME(223,1)=0; MDME(224,1)=0; MDME(225,1)=0; MDME(226,1)=0; MSTJ(28)=0; PMAS(25,1)=125.; PMAS(25,2)=0.4143E-02; MSTJ(41)=2; MSTU(22)=2000; PARJ(21)=0.40000; PARJ(41)=0.11000; PARJ(42)=0.52000; PARJ(81)=0.25000; PARJ(82)=1.90000; MSTJ(11)=3; PARJ(54)=-0.03100; PARJ(55)=-0.00200; PARJ(1)=0.08500; PARJ(3)=0.45000; PARJ(4)=0.02500; PARJ(2)=0.31000; PARJ(11)=0.60000; PARJ(12)=0.40000; PARJ(13)=0.72000; PARJ(14)=0.43000; PARJ(15)=0.08000; PARJ(16)=0.08000; PARJ(17)=0.17000; MSTP(3)=1;MSTP(71)=1;  MSTP(151)=1;  MSTJ(22)=4; PARJ(73)=2250; PARJ(74)=2500"
```

Comment:
```suggestion
$ps_PYTHIA_PYGIVE = "MDME(210,1)=0; MDME(211,1)=0; MDME(212,1)=0; MDME(213,1)=0; MDME(214,1)=1; MDME(215,1)=0; MDME(218,1)=0; MDME(219,1)=0; MDME(220,1)=0; MDME(222,1)=0; MDME(223,1)=0; MDME(224,1)=0; MDME(225,1)=0; MDME(226,1)=0; MSTJ(28)=0; PMAS(25,1)=125.; PMAS(25,2)=0.4143E-02; MSTJ(41)=2; MSTU(22)=2000; PARJ(21)=0.40000; PARJ(41)=0.11000; PARJ(42)=0.52000; PARJ(81)=0.25000; PARJ(82)=1.90000; MSTJ(11)=3; PARJ(54)=-0.03100; PARJ(55)=-0.00200; PARJ(1)=0.08500; PARJ(3)=0.45000; PARJ(4)=0.02500; PARJ(2)=0.31000; PARJ(11)=0.60000; PARJ(12)=0.40000; PARJ(13)=0.72000; PARJ(14)=0.43000; PARJ(15)=0.08000; PARJ(16)=0.08000; PARJ(17)=0.17000; MSTP(3)=1;MSTP(71)=1;  MSTP(151)=0;  MSTJ(22)=4; PARJ(73)=2250; PARJ(74)=2500"
```

This (`MSTP(151)=0`) turns** off the vertex smearing in pythia, which is probably better than just leaving the smearing parameters at their defaults